<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_dataHistogram</class>
 <widget class="QDialog" name="Dialog_dataHistogram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Гистограмма распределения</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox{
	border: 1px solid grey;
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>511</height>
    </rect>
   </property>
   <property name="title">
    <string>Параметры гистограммы</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>181</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип разделения</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_intervalsType">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>191</width>
      <height>37</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_genIntervals">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>460</y>
      <width>103</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Построить</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>221</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_histogratorParams"/>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>561</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_histogram"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
